### PR TITLE
Moved the locale to a path param

### PR DESCRIPTION
**Pull request by Chris-V**
The extraction of the path param is archaic. I think it would be overkill to use something like restlet to do it for us though.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,7 +11,7 @@
     <name>Arcbees Website</name>
 
     <properties>
-        <target.jdk>1.7</target.jdk>
+        <target.jdk>${velocity.version}</target.jdk>
```
Comment (meriouma) [inline pom.xml]: Nope! ;) Revert to 1.7

Comment (Chris-V) [inline pom.xml]: Whoops

File: src/main/java/com/arcbees/website/server/HomeServlet.java [hunk 2]
```diff
@@ -1,17 +1,88 @@
...
+    private String extractLocale(HttpServletRequest request) {
+        String path = Strings.nullToEmpty(request.getRequestURI());
+        Matcher matcher = EXTRACT_LOCALE_PATTERN.matcher(path);
+
+        String locale = null;
+        if (matcher.find()) {
+            locale = matcher.group(1);
+        }
+
+        return Strings.isNullOrEmpty(locale) ? DEFAULT_LOCALE : locale;
```
Comment (meriouma) [inline src/main/java/com/arcbees/website/server/HomeServlet.java]: You should include a check for `Accept-Language` header and maybe later we could also store the user's selected locale in a cookie

**Comment by meriouma** [comment]
LGTM

File: src/main/java/com/arcbees/website/server/HomeServlet.java (hunk 2 above)
Comment (Chris-V) [inline src/main/java/com/arcbees/website/server/HomeServlet.java]: There you go! The different variables are (in order of importance): path param, cookie, `Accept-Language`, "en".

**Comment by christiangoudreau** [comment]
LGTM

File: src/main/webapp/WEB-INF/appengine-web.xml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
-    <application>arcbeeswebsite</application>
+    <application>arcbees-temp-site</application>
```
Comment (christiangoudreau) [inline src/main/webapp/WEB-INF/appengine-web.xml]: You'll need to change that back

**Pull request merged**